### PR TITLE
Add Docker instructions in tensorboard-in-notebooks

### PR DESCRIPTION
**Pull request by brunapinos**
Hi!
As described in #1952, it was necessary to add the information to expose the TensorBoard port when running the notebook using TensorFlow docker image. So, I added a paragraph in tensorboard-in-notebooks file, explaining the usage with docker, in **Setup** section:

![Screenshot of new “Setup” section, as rendered in Colab][1]

[1]: https://user-images.githubusercontent.com/4317806/59947291-b4d1f100-9421-11e9-852e-0bc322fb69c0.png

### CODE REVIEW

**Comment by nfelt** [comment]
@brunapinos Thanks for the contribution!  ReviewNB can't render the diff, is it possible the new .ipynb file isn't quite formatted right?  Also, is there a chance you might be able to reduce the size of the diff in the raw .ipynb?  It looks like a lot of extra lines were moved around (indentation changes, re-ordering json fields) even though I believe content-wise you just added the one paragraph.

**Comment by brunapinos** [comment]
Thanks for the review! I'll reduce it!

**Comment by brunapinos** [comment]
@nfelt  So, I had to give a push force to commit my changes with Colab. I don't know if it was the best way to do that. If you want, I can open another PR to commit these changes.

**Comment by brunapinos** [comment]
Now the diff is only referred to the paragraph added!

File: docs/r2/tensorboard_in_notebooks.ipynb [hunk 1]
```diff
@@ -110,6 +110,25 @@
...
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "In case you are running a [Docker](https://docs.docker.com/install/) image of [Jupyter Notebook server using TensorFlow's nightly](https://www.tensorflow.org/install/docker#examples_using_cpu-only_images), it is necessary to expose not only the notebook's port, but the TensorBoard's port.\n",
+        "\n",
+        "Thus, run the container with the following command:\n",
+        "\n",
+        "```\n",
+        "docker run -it -p 8888:8888 -p 6006 \\\n",
```
Comment (wchargin) [inline docs/r2/tensorboard_in_notebooks.ipynb]: This doesn’t seem to work for me—when I use `%tensorboard` in the
Jupyter notebook, I still get “localhost refused to connect”. It works
if I change `-p 6006` to `-p 6006:6006`. Can/should we use that instead?

(I’m not super familiar with Docker, so if I’m missing something then
please fill me in!)

Comment (brunapinos) [reply docs/r2/tensorboard_in_notebooks.ipynb]: Sorry, it was really missing ` :6006` in command. I've already fixed.

**Pull request merged**